db/migrate/20230118195424_add_user_ref_likes.rb

```diff
@@ -0,0 +1,4 @@
+class AddUserRefLikes < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```

Comment:
- [ ] Kindly reference the user in the likes table, currently this migration is empty, also make sure to use the keyword(auther_id) to make the reference. hint :point_down: 

```rb

class AddUserRefLikes < ActiveRecord::Migration[7.0]
  def change
    add_reference :likes, :author, null: false, foreign_key: { to_table: 'users' }
  end
end
```